packages/lwc/index.d.ts

```diff
@@ -176,6 +176,7 @@ declare module 'lwc' {
 
         readonly template: ShadowRootTheGoodPart;
         readonly shadowRoot: null;
+        readonly refs: { [key: string]: Element | LightningElement };
```

Comment:
It's odd to me that `LightningElement` is not a subclass of `Element`, but I can see that it's not right now... Might be worth looking into later.

Reply:
@nolanlawson That's been a common complaint--we have planned work to address that this release.